homeassistant/auth/mfa_modules/notify.py

```diff
@@ -306,7 +318,9 @@ async def async_step_init(
         schema = vol.Schema(
             {
                 vol.Required("notify_service"): vol.In(self._available_notify_services),
-                vol.Optional("target"): str,
+                vol.Optional("target"): selector.EntitySelector(
```

Comment:
this prevents setting a target for the legacy service, right? the old target can be any string, not just entities.